deploy/README.md

```diff
@@ -0,0 +1,309 @@
+   `template.env` above. NOTE: Do not change `DB_*` values unless you are also
+   re-creating the database ([see below](#removing-wikibase-suite-completely-with-all-its-data).
+2. Delete your LocalSettings.php from the ./config directory.
+3. Remove and re-create containers with:
+
+```sh
+docker compose down
+docker compose up --wait
+```
+
+### Advanced Configuration
+On first launch, WBS Deployment Kit will create files in the `./config` directory next to your `.env` file and the `docker-compose.yml` and `template.env`. This is the configuration of your instance. **You own those files.** Be sure to include them in your backups.
+
+#### `config/LocalSettings.php`
+This file is generated by the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php) and augmented by the Wikibase containers `entrypoint.sh` on first launch. Once this file has been generated, you own it. This means you can make changes, you even might need to make changes on major version updates (as describes in MediaWiki UPGRADE notes). In order to ease keeping track of your own adjustments vs settings generated by the Wikibase container, we provide a file called `LocalSettings.override.php`. This file is the preferred location for your adjustments. See below.
+
+The absence of `LocalSettings.php` is the trigger for the Wikibase container to run the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php). If you need to run the installer again, remove the generated `LocalSettings.php` file and restart your instance.
+
+#### `config/LocalSettings.override.php`
+This file is the location for your MediaWiki and Wikibase customizations. If you need to regenerate your `LocalSettings.php`, your `LocalSettings.override.php` is the safe place that is never rewritten, even when running the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php).
+
+If you want to, for example, change your site name, just put the following in your `LocalSettings.override.php`.
+```php
+<?php
+$wgSitename = "My New Site Name";
+```
+
+Restart your Wikibase container with 
+```sh
+docker restart wikibase-suite-wikibase-1
+```
+
+#### `config/wikibase-php.ini`
+This file is a `php.ini` override file, a good place to tune PHP configuration values. It will be loaded by the Wikibase Webservers PHP interpreter.
+
+#### docker-compose.yml
+To further customize your instance, consider adjusting `docker-compose.yml`. In order to ease updating to newer versions of the WBS Deployment Kit, consider putting your customization into a new file called `docker-compose.override.yml` and from now on, restart your instance with
+
+```sh
+docker compose -f docker-compose.yml -f docker-compose.override.yml down
+docker compose -f docker-compose.yml -f docker-compose.override.yml up --wait
+```
+
+This way your changes are kept separate from the original WBS Deploying Kit code.
+
+### Managing your Data
```

Comment:
Nitpick and a matter of opinion for sure, but I wouldn't capitalise "Data" here or below.

Reply:
went back and forth on that. trusting your guts. done.

Reply:
In general I'm also always am back and forth re Title Case in docs. But in this instance I'm flagging it as Dan recently pointed out to me that the MediaWiki way is _never_ title case.   I may have gotten that wrong, and it certainly does that need to apply to our code comments, but may come-up when we look into how to sync the README file with the MediaWiki Wikibase Docker install page...